tests/integration/interaction/test_responsive.py

```diff
@@ -156,3 +162,132 @@ def test_responsive_plot_starts_at_correct_size(output_file_url, selenium):
     # Canvas width should be just under 1000
     assert canvas.size['width'] > 900
     assert canvas.size['width'] < 1000
+
+
+def test_responsive_resizes_width_and_height(output_file_url, selenium):
```

Comment:
@birdsarah Thanks for this mini "framework", it was real easy to add tests for it using your examples!